### PR TITLE
IoT Cloud Security Article

### PR DESCRIPTION
**Pull request by karlsoderby**
## What This PR Changes
- Article from Federicio De Meo in markdown format, added to the IOT Cloud Feature section. 

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/arduino-cloud/02.features/00.security-considerations/security-considerations.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+### Arduino Cloud CLI
+
+The Arduino Cloud CLI provides a way for interacting with the Arduino IoT Cloud via the command line. The communication is carried out via an HTTPS connection and authentication is performed via API key credentials stored on the local machine.
+
+### Arduino Cloud Editor
+
+The Arduino Cloud Editor allows customers to create sketches they can subsequently deploy on boards. Each customer has a separate area where their sketches are stored so that no other customer has access to them. Sketches can be made publicly available to everyone, in which case access is granted by whomever possesses the link to access them.
+
+The Arduino Cloud Editor also provides a secret tab that can be used to securely store sensitive information needed by the device (e.g. WiFi credentials, API credentials). Secrets in this tab are encrypted with the AES-256 algorithm to safely store them in the Cloud.
```
Comment (jhansson-ard) [inline content/arduino-cloud/02.features/00.security-considerations/security-considerations.md:132]:
```suggestion
The Arduino Cloud Editor also provides a secret tab that can be used to securely store sensitive information needed by the device (e.g. Wi-Fi credentials, API credentials). Secrets in this tab are encrypted with the AES-256 algorithm to safely store them in the Cloud.
```

File: content/arduino-cloud/02.features/00.security-considerations/security-considerations.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+Devices onboarding and communication with the Arduino Cloud is ensured to protect the confidentiality of the data exchanged between Arduino devices and the Cloud. The following image shows the Device setup phase and the Device operation phase.
+
+![Device Setup & Operation Phases.](assets/device-phases.png)
+
+> **Note**: this communication process is available only on devices equipped with a secure element chip.
+
+**Device setup phase**: 
+* The setup phase is initiated by the web browser with a secure connection to the Arduino IoT Cloud over HTTPS requesting to create a new device.
+* The IoTCloud provides a special sketch that is run on the device to generate:
```
Comment (jhansson-ard) [inline content/arduino-cloud/02.features/00.security-considerations/security-considerations.md:151]: Is this supposed to be "IoT Cloud" @rhaidiz? With a space in between `IoT` and `Cloud`´?

Comment (rhaidiz) [reply content/arduino-cloud/02.features/00.security-considerations/security-considerations.md:151]: I believe so as we write it that way also in the [docs](https://docs.arduino.cc/arduino-cloud/).

Comment (jhansson-ard) [inline content/arduino-cloud/02.features/00.security-considerations/security-considerations.md:151]:
```suggestion
* The IoT Cloud provides a special sketch that is run on the device to generate:
```

**Pull request merged**